docs/marionette.region.md

```diff
@@ -267,16 +267,12 @@ The default implementation of `attachHtml` works a bit like the following:
 var Mn = require('backbone.marionette');
 
 Mn.Region.prototype.attachHtml = function(view){
-  if (this.isShowingAView()) {
-    this.removeRenderedView();
-  }
-  this.$el.append(view.el);
+  this.el.appendChild(view.el);
 }
 ```
 
-If there's a view being displayed, this code replaces it and renders then
-attaches the new view's HTML. You can override `attachHtml` for transition
-effects and more.
+The existing `attachHtml` takes an extra `shouldReplace` argument, however if
```

Comment:
Yeah.. that `shouldReplace` argument was once a `_shouldReplace` indicating it wasn't for public consumption..  I'm really thinking we want to pull the replaceEl stuff out of here..  I'm not convinced that overriding `attachHtml` should _break_ `replaceElement`.  I think we should move that flag and check outside of this function.  So I'm worried that documenting this argument would make changing it breaking.